api/cards/filterCards.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
+import { filterKnownCards, getAllCards } from "~/utils/cards";
+
+const knownTags: any = {};
+
+export default (_: Request, response: Response) =>
+  getAllCards()
+    .then(cards => filterKnownCards(cards))
+    .then(cards => {
+      cards.forEach(card => {
+        card.tags.forEach(tag => {
+          if (!knownTags[tag]) {
+            knownTags[tag] = 1;
+          } else {
+            knownTags[tag] += 1;
+          }
+        });
+      });
+    })
+    .then(() =>
+      response.send({
+        apiName: "cards/filter",
+        tags: knownTags
+      })
+    )
+    .catch((e: Error) =>
+      response.send({
+        apiName: "cards/filterCards",
+        message: e.message,
+        stack: e.stack
+      })
+    );
```

Comment:
It may be a strange idea to create endpoint, which returns all the tags, because it runs all over the cards to collect this info. Do you have any ideas how to avoid this?

Reply:
for me that's fine, just iterate over cards once, not on every request